testing/testcafe/tests/editors/list/common.ts

```diff
@@ -314,9 +314,9 @@ test('Disabled item should not have focus (T815151)', async (t) => {
     .click(searchInput)
     .pressKey('tab')
     .expect(firstItem.isFocused)
-    .notOk()
```

Comment:
We updated asserts but didn't updated the test name. Test name seems to be incorrect now. 
I suggest to rename this way:
```javascript
test('Disabled item should be focused on tab press to match accessibility criteria', async (t) => {
```

Reply:
A nice suggestion! Thank you a lot! Was fixed here [25bf9f775185867becc632f8384161cc5d472868].